appveyor.yml

```diff
@@ -11,7 +11,7 @@ environment:
       VCVARSALL: "C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Auxiliary/Build/vcvarsall.bat"
       PLATFORM:  x64
 
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015  # used as mingw
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019  # used as mingw
```

Comment:
Ah, I was testing with AppVeyor since its MinGW version has internal compile errors when using `<filesystem>`. I haven't touched the config since then though since I don't know how to proceed.